engine/maptile.py

```diff
@@ -38,3 +38,11 @@ def reveal_npcs(self):
     
     def greet_npcs(self):
         return [npc.greet() for npc in self.local_npcs]
+
+    def pickupitem(self, description):
+        item_index = 0
+        for item in self.local_items:
+            if item.description == description:
+                return self.local_items.pop(item_index)
+            item_index += 1
+        return "You cannot get that."
```

Comment:
In general this is something unique about Python - you can return multiple different types from a single function. However, it's a confusing practice. I'd recommend you return None here if you can't find the item, and have the caller of this function generate this string when you return None.